app/messaging/crm/create-crm-message.js

```diff
@@ -1,10 +1,12 @@
 const schema = require('./crm-schema')
+const { ffcApiPath, statementReceiverEndpoint } = require('../../config')
 
-const createCrmMessage = (statement, blobUrl) => {
+const createCrmMessage = (statement, blobUrl, filename) => {
   const crm = {
-    sbi: statement.sbi,
+    apiLink: `${statementReceiverEndpoint}/${ffcApiPath}/statement?filename=${filename}`,
```

Comment:
Only part of the path is used within the variable, `/statement` and the query string are outside.  Should we include all but the query string, or the query string too?

What about changing the API to be `/statementReceiverEndpoint/ffcApiPath/statement`  rather than needing a querystring?